diff --git a/node_modules/@types/node-cron/index.d.ts b/node_modules/@types/node-cron/index.d.ts
index a3aa237..21a74bb 100644
--- a/node_modules/@types/node-cron/index.d.ts
+++ b/node_modules/@types/node-cron/index.d.ts
@@ -8,7 +8,7 @@ import { EventEmitter } from "events";
  */
 export function schedule(
     cronExpression: string,
-    func: ((now: Date | "manual" | "init") => void) | string,
+    func: ((now: Date | "manual" | "init") => void),
     options?: ScheduleOptions,
 ): ScheduledTask;
 
