diff --git a/node_modules/node-cron/src/background-scheduled-task/daemon.js b/node_modules/node-cron/src/background-scheduled-task/daemon.js
index 022eb69..e530c01 100644
--- a/node_modules/node-cron/src/background-scheduled-task/daemon.js
+++ b/node_modules/node-cron/src/background-scheduled-task/daemon.js
@@ -1,4 +1,4 @@
-const ScheduledTask = require('../scheduled-task');
+const ScheduledTask = require('node-cron/src/scheduled-task');
 
 let scheduledTask;
 
diff --git a/node_modules/node-cron/src/convert-expression/index.js b/node_modules/node-cron/src/convert-expression/index.js
index c1a992d..37728ec 100644
--- a/node_modules/node-cron/src/convert-expression/index.js
+++ b/node_modules/node-cron/src/convert-expression/index.js
@@ -1,10 +1,10 @@
 'use strict';
 
-const monthNamesConversion = require('./month-names-conversion');
-const weekDayNamesConversion = require('./week-day-names-conversion');
-const convertAsterisksToRanges = require('./asterisk-to-range-conversion');
-const convertRanges = require('./range-conversion');
-const convertSteps = require('./step-values-conversion');
+const monthNamesConversion = require('node-cron/src/convert-expression/month-names-conversion');
+const weekDayNamesConversion = require('node-cron/src/convert-expression/week-day-names-conversion');
+const convertAsterisksToRanges = require('node-cron/src/convert-expression/asterisk-to-range-conversion');
+const convertRanges = require('node-cron/src/convert-expression/range-conversion');
+const convertSteps = require('node-cron/src/convert-expression/step-values-conversion');
 
 module.exports = (() => {
 
diff --git a/node_modules/node-cron/src/node-cron.js b/node_modules/node-cron/src/node-cron.js
index 459f161..0c0c4bd 100644
--- a/node_modules/node-cron/src/node-cron.js
+++ b/node_modules/node-cron/src/node-cron.js
@@ -1,9 +1,8 @@
 'use strict';
 
-const ScheduledTask = require('./scheduled-task');
-const BackgroundScheduledTask = require('./background-scheduled-task');
-const validation = require('./pattern-validation');
-const storage = require('./storage');
+const ScheduledTask = require('node-cron/src/scheduled-task');
+const validation = require('node-cron/src/pattern-validation');
+const storage = require('node-cron/src/storage');
 
 /**
  * @typedef {Object} CronScheduleOptions
@@ -30,9 +29,6 @@ function schedule(expression, func, options) {
 }
 
 function createTask(expression, func, options) {
-    if (typeof func === 'string')
-        return new BackgroundScheduledTask(expression, func, options);
-
     return new ScheduledTask(expression, func, options);
 }
 
diff --git a/node_modules/node-cron/src/pattern-validation.js b/node_modules/node-cron/src/pattern-validation.js
index 5bfca83..347882d 100644
--- a/node_modules/node-cron/src/pattern-validation.js
+++ b/node_modules/node-cron/src/pattern-validation.js
@@ -1,6 +1,6 @@
 'use strict';
 
-const convertExpression = require('./convert-expression');
+const convertExpression = require('node-cron/src/convert-expression');
 
 const validationRegex = /^(?:\d+|\*|\*\/\d+)$/;
 
diff --git a/node_modules/node-cron/src/scheduled-task.js b/node_modules/node-cron/src/scheduled-task.js
index 52a1f59..ba7cfa5 100644
--- a/node_modules/node-cron/src/scheduled-task.js
+++ b/node_modules/node-cron/src/scheduled-task.js
@@ -1,8 +1,8 @@
 'use strict';
 
 const EventEmitter = require('events');
-const Task = require('./task');
-const Scheduler = require('./scheduler');
+const Task = require('node-cron/src/task');
+const Scheduler = require('node-cron/src/scheduler');
 const uuid = require('uuid');
 
 class ScheduledTask extends EventEmitter {
diff --git a/node_modules/node-cron/src/scheduler.js b/node_modules/node-cron/src/scheduler.js
index 72febb4..53012f2 100644
--- a/node_modules/node-cron/src/scheduler.js
+++ b/node_modules/node-cron/src/scheduler.js
@@ -1,7 +1,7 @@
 'use strict';
 
 const EventEmitter = require('events');
-const TimeMatcher = require('./time-matcher');
+const TimeMatcher = require('node-cron/src/time-matcher');
 
 class Scheduler extends EventEmitter{
     constructor(pattern, timezone, autorecover){
diff --git a/node_modules/node-cron/src/time-matcher.js b/node_modules/node-cron/src/time-matcher.js
index 666a9e3..5647d90 100644
--- a/node_modules/node-cron/src/time-matcher.js
+++ b/node_modules/node-cron/src/time-matcher.js
@@ -1,5 +1,5 @@
-const validatePattern = require('./pattern-validation');
-const convertExpression = require('./convert-expression');
+const validatePattern = require('node-cron/src/pattern-validation');
+const convertExpression = require('node-cron/src/convert-expression');
 
 function matchPattern(pattern, value){
     if( pattern.indexOf(',') !== -1 ){
diff --git a/node_modules/node-cron/test/task-test.js b/node_modules/node-cron/test/task-test.js
index 0f866e2..44a5aba 100644
--- a/node_modules/node-cron/test/task-test.js
+++ b/node_modules/node-cron/test/task-test.js
@@ -1,6 +1,6 @@
 const { assert } = require('chai');
 const sinon = require('sinon');
-const Task = require('../src/task');
+const Task = require('node-cron/src/task');
 
 describe('Task', () => {
     beforeEach(() => {
diff --git a/node_modules/node-cron/test/time-matcher-test.js b/node_modules/node-cron/test/time-matcher-test.js
index f7b5563..543ba3f 100644
--- a/node_modules/node-cron/test/time-matcher-test.js
+++ b/node_modules/node-cron/test/time-matcher-test.js
@@ -1,5 +1,5 @@
 const { assert } = require('chai');
-const TimeMatcher = require('../src/time-matcher');
+const TimeMatcher = require('node-cron/src/time-matcher');
 const moment = require('moment-timezone');
 
 describe('TimeMatcher', () => {
